spec/rails_helper.rb

```diff
@@ -45,6 +46,14 @@
   # instead of true.
   config.use_transactional_fixtures = true
 
+  config.before do
+    # Reset this to the default so you don't need to clean up specs that want
+    # inline jobs.
+    Sidekiq::Testing.fake!
```

Comment:
People generally forget to clean this up, I like to enforce it

Reply:
@hiattp handy 👍